schedule/systemd-boot/sdboot-fde.yaml

```diff
@@ -0,0 +1,9 @@
+---
+name: gnome-sdboot-fde
+description: >
+    Maintainer: Santiago Zarate <santiago.zarate@suse.com>
+    Download systemd tar file and create a hdd
+schedule:
+    - boot/boot_to_desktop
+    - console/sdbootutil_enroll
+    - shutdown/shutdown
```

Comment:
IMO this should also do some other tests afterwards to ensure it keeps working for new snapshots as well